src/components/admin/Login.vue

```diff
@@ -25,37 +24,39 @@
 </template>
 <script>
 
-    import Dialog from "../common/Dialog";
 
     export default {
         name: 'login',
-        components: {Dialog},
+        components: {
+
+        },
         data() {
-            return {}
+            return {
+                login: '',
+                password: ''
+            }
         },
         methods: {
             handleSubmit() {
-                const login = document.getElementById('username').value;
-                const password = document.getElementById('password').value;
-                const authorization = `Basic ${btoa(`${login}:${password}`)}`;
-                fetch("http://jakas.pl"/*Todo endpoint from backend*/, {
+                const authorization = `Basic ${btoa(`${this.login}:${this.password}`)}`;
```

Comment:
To powinny być ukryte w klasce do robienia zapytań. Należy w App.vue stworzyć jej obiekt, eksportować i tutaj używać